ci-operator/step-registry/kueue-operator/install/operator/kueue-operator-install-operator-commands.sh

```diff
@@ -0,0 +1,168 @@
+        echo "No cert-manager PackageManifest found. Skipping cert-manager installation..."
+    else
+        echo "Installing cert-manager operator..."
+        oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cert-manager-operator
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cert-manager-operator-og
+  namespace: cert-manager-operator
+spec:
+  targetNamespaces: []
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-cert-manager-operator
+  namespace: cert-manager-operator
+spec:
+  channel: stable-v1
+  name: openshift-cert-manager-operator
+  source: $CATSRC_NAME
+  sourceNamespace: openshift-marketplace
+EOF
+
+        if wait_for_state "deployment/cert-manager-operator-controller-manager" "condition=Available" "5m" "cert-manager-operator"; then
+            echo "Cert-manager operator is ready"
+        else
+            echo "Timed out waiting for cert-manager operator. Dumping resources for debugging..."
+            run_command "oc get pod -n cert-manager-operator"
+            run_command "oc get event -n cert-manager-operator"
+            exit 1
+        fi
+    fi
+
+    # Check for kueue operator
+    kueue_output=$(oc get packagemanifest -n openshift-marketplace -l=catalog=$CATSRC_NAME --field-selector=metadata.name=kueue-operator 2>&1)
+    if [[ $? -ne 0 ]] || echo "$kueue_output" | grep -q "No resources found"; then
+        echo "No kueue PackageManifest found. Skipping kueue installation..."
+        exit 0
+    fi
+
+    if [[ "$TARGET_NAMESPACES" == "!all" ]]; then
+        TARGET_NAMESPACES=""
+    fi
+
+    echo "Creating the Namespace, OperatorGroup and Subscription for the kueue operator installation..."
+    oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-kueue-operator
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: kueue-operator-group
+  namespace: openshift-kueue-operator
+spec:
+  targetNamespaces: [$TARGET_NAMESPACES]
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: kueue-operator
+  namespace: openshift-kueue-operator
+spec:
+  channel: $CHANNEL
+  name: kueue-operator
+  source: $CATSRC_NAME
+  sourceNamespace: openshift-marketplace
+EOF
+
+    if wait_for_state "deployment/openshift-kueue-operator" "condition=Available" "5m" "openshift-kueue-operator"; then
+        echo "Kueue operator is ready"        
+        installedCSV=$(oc get subscription kueue-operator -n openshift-kueue-operator -o jsonpath='{.status.installedCSV}')
+        run_command "oc get csv $installedCSV -n openshift-kueue-operator"
+        run_command "oc get csv $installedCSV -n openshift-kueue-operator -o=jsonpath='{.spec.relatedImages}'"
+        echo
+    else
+        echo "Timed out after 5m. Dumping resources for debugging..."
+        run_command "oc get pod -n openshift-kueue-operator"
```

Comment:
we could collect this information with the must-gather.
I have the following PR to facilitate re-using the post step with must-gather and clean-up https://github.com/openshift/release/pull/66799 if you're interested in having a look,

Reply:
sure that makes sense. Would you like to cover this as part of your PR?